cypress/support/pages/course/pageCommonCourses.js

```diff
@@ -69,5 +77,50 @@ class Courses_Common {
   canNotAddBigBlueButton() {
     cy.get(Courses_Common.#toolsList).should('not.exist')
   }
+
+  clickOnCreateCourseFAB() {
+    cy.get(Courses_Common.#createCourse).click()
+  }
+
+  clickOnCreateContentFAB() {
```

Comment:
we can specify 'task' in this method name because 'topic' is also part of content, but this method is only to create  a new task right ? --> e.g. 'clickOnCreateTaskFAB()